NuRadioReco/modules/io/RNO_G/readRNOGDataMattak.py

```diff
@@ -556,17 +563,18 @@ def _filter_event(self, evtinfo, event_idx=None):
         -------
 
         skip: bool
-            Returns True to skip/reject event, return False to keep/read event
+            Returns False to skip/reject event, return True to keep/read event
```

Comment:
Nice catch! Do you think we should slightly rename the function to better describe this change. For example: `_select_events`